site/src/components/HelpTooltip/HelpTooltip.tsx

```diff
@@ -242,7 +240,7 @@ const styles = {
     marginBottom: theme.spacing(1),
     color: theme.palette.text.primary,
     fontSize: 14,
-    lineHeight: "120%",
+    lineHeight: "150%",
```

Comment:
Not sure what you mean by "work better with CSS inheritance". `150%` and `1.5` mean the same thing in CSS. 😅 and I prefer percent, because the unit makes it clearer what is happening imo.

Reply:
I wish they were the same, but [they aren't](https://stackoverflow.com/a/12304722)

Reply:
why is CSS like this 😭